unit-tests/shared/src/test/scala/javalib/net/InetAddressTest.scala

```diff
@@ -40,6 +42,12 @@ class InetAddressTest {
     assertFalse(all == null)
     assertTrue(all.length >= 1)
 
+    if (!Platform.isWindows) {
+      // TODO remove filter on main
+      for (alias <- all; if alias.isInstanceOf[Inet4Address])
```

Comment:
When I ported that line of code to my IPv6 WIP PR, I left that filter in place,
because it made sense. (i.e. was the right way to do it, in my estimation & experience).
I removed the TODO comment entirely.

I recommend that this TODO line be committed as is, and let my IPv6 work remove it,
or that it be removed entirely. Dealer's choice.